lib/improver/ensemble_copula_coupling/ensemble_copula_coupling_utilities.py

```diff
@@ -161,11 +163,9 @@ def create_cube_with_percentiles(percentiles, template_cube, cube_data,
             template cube.
 
     """
-    percentile_coord_name = custom_name or 'percentile'
-
     percentile_coord = iris.coords.DimCoord(
-        np.float32(percentiles), long_name=percentile_coord_name,
-        units=unit.Unit("%"), var_name=percentile_coord_name)
+        np.float32(percentiles), long_name='percentile',
+        units=unit.Unit("%"), var_name='percentile')
```

Comment:
You can also remove "custom_name" as an argument to this function...

Reply:
I thought I had, but I'll do that now.